lib/vsc/install/ci.py

```diff
@@ -248,7 +248,7 @@ def parse_vsc_ci_cfg():
             cfgparser.read(VSC_CI_INI)
             cfgparser.items(VSC_CI)  # just to make sure vsc-ci section is there
         except (configparser.NoSectionError, configparser.ParsingError) as err:
-            logging.error(f"ERROR: Failed to parse {VSC_CI_INI}: {err}")
+            logging.error("ERROR: Failed to parse %s: %s", VSC_CI_INI, err)
```

Comment:
why did you revert here?

Reply:
so for logging pylint prefers lazy logging, for anything else f-strings.

Reply:
I'm not sure what my opinion about that is.

Reply:
it's correct, but not really needed for error, as that is always logged.